cmd/utils/cmd.go

```diff
@@ -171,9 +175,10 @@ func hasAllBlocks(chain *core.BlockChain, bs []*types.Block) bool {
 	return true
 }
 
-func ExportChain(blockchain *core.BlockChain, fn string) error {
+func ExportChain(blockchain *core.BlockChain, chainDb ethdb.Database, fn string, first uint64, last uint64) error {
 	log.Info("Exporting blockchain", "file", fn)
-	fh, err := os.OpenFile(fn, os.O_CREATE|os.O_WRONLY|os.O_TRUNC, os.ModePerm)
+	// TODO verify mode perms
+	fh, err := os.OpenFile(fn, os.O_CREATE|os.O_APPEND|os.O_WRONLY, os.ModePerm)
```

Comment:
I don't know where that TODO came from; maybe I'm operating on an old version of the code?